### PR TITLE
Feature/more columns

### PR DESCRIPTION
**Pull request by danielkopo**

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def protected():
             'country': '1',
         },
         {
-            'name': 'Banana Farm',
+            # 'name': 'Banana Farm',
```
Comment (domerd) [inline backend/main.py:38]: By mistake?

File: src/FarmerTable.js [hunk 2]
```diff
@@ -38,7 +52,7 @@ const FarmerTable = () => {
                 <PathForm
                     deliveryId={record.id}
                     sourceFarmer={record.source}
-                    key={record.id}
+                    deliveryID={record.id}
```
Comment (domerd) [inline src/FarmerTable.js]: Already exists in line 53

File: src/FarmerTable.js [hunk 3]
```diff
@@ -1,31 +1,45 @@
...
         {
-            title: 'Source Farmer',
+            title: 'Delivery Name',
+            dataIndex: 'name',
+            key: 'name',
+            render: (text, record) => (
+                <Input
+                    deliveryId={record.id}
+                    defaultValue={text}
+                    placeholder={text ? false : 'Insert name'}
```
Comment (domerd) [inline src/FarmerTable.js]: I think that if a value exists you don't need to turn off the placeholder:
```JS
placeholder="Delivery Name"
```

**Review by domerd — COMMENTED** [review]
Open question:
Should we add the name even if we are only changing the path (for clearance)?
Now, only if the user changes the name it will appear in the result JSON.

What do you think?

File: backend/main.py (hunk 1 above)
Comment (danielkopo) [reply backend/main.py:38]: Nope, doesn't exist in order to show up the placeholder

Comment (domerd) [reply backend/main.py:38]: Cool!

**Review by domerd — APPROVED** [review]

**Pull request merged**